imagedephi/redact/svs.py

```diff
@@ -16,7 +22,19 @@
 
 class SvsDescription:
     prefix: str
-    metadata: dict[str, str]
+    metadata: dict[str, str | int | float]
+
+    def try_get_numeric_value(self, value: str) -> str | int | float:
+        """Given an ImageDescription value, return a number version of it if applicable."""
+        try:
+            int(value)
+            return int(value)
+        except ValueError:
+            try:
+                float(value)
+                return float(value)
+            except ValueError:
+                return value
```

Comment:
Since the `ImageDescription` of a tiff is a string, and we want to be able to check the types of values for `check_type` rules, I think we'll need to convert values if applicable at this stage.